ckan/controllers/group.py

```diff
@@ -227,15 +240,15 @@ def _read(self, id, limit):
         def search_url(params):
             if group_type == 'organization':
```

Comment:
IGroupForm.group_types lets plugins define different types for their orgs. I take that to mean is_organization is the definitive test. I haven't actually tried creating different org types, but that's what I hope to do with scheming in the future: make orgs as customizable as datasets, and one day maybe users too.